rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
 postgresql_password: "{{ lookup('password', '/tmp/pass chars=ascii_letters,digits') }}"
+postgresql_user: "admin"
```

Comment:
I think everything but these should go. The rest will come from extra vars and you'll just specify the default size in the apb.yml the same as for the other parameters.

postgresql_password: "{{ lookup('password', '/tmp/pass chars=ascii_letters,digits') }}"
image: "postgresql-{{ postgresql_version | replace('.', '') }}-rhel7"
state: present